<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>이웃 기어가기와 글자 구름</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>700</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>네이버 지식인 크로울링</string>
     </attribute>
     <widget class="QLineEdit" name="ed_search">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
     <widget class="QLabel" name="naver_search">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>네이버검색</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameofsavefile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>저장파일명</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_result">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_close">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>610</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>닫기</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>글자 구름</string>
     </attribute>
     <widget class="QLabel" name="file_choose">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>파일 선택</string>
      </property>
     </widget>
     <widget class="QLabel" name="forbidden_word">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>금지어</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_file">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_stopword">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_process">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>분석</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_close_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>610</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>닫기</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_word">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>671</width>
        <height>503</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
